package.json

```diff
@@ -103,6 +103,7 @@
     "taskcluster-lib-scopes": "link:libraries/scopes",
     "taskcluster-lib-urls": "^12.0.0",
     "taskcluster-lib-validate": "link:libraries/validate",
+    "taskcluster-worker-manager": "link:services/worker-manager",
```

Comment:
Why does this need to be added here?

Reply:
question for a question: why are other taskcluster dependencies added here?

Reply:
the taskcluster bits added here are libraries that are used in the services or other libraries. Services aren't imported by any other code so shouldn't be in here.

Reply:
https://github.com/taskcluster/taskcluster/pull/843/files#diff-7b7f521247a2c870baae56e97f99da7eR1 (sorry I was assuming you read the code since you had approved it)